packages/astro/src/core/build/static-build.ts

```diff
@@ -231,7 +231,7 @@ async function clientBuild(
 				output: {
 					format: 'esm',
 					entryFileNames: '[name].[hash].js',
```

Comment:
Quick question: Why aren't entry files moved into `assets` as well?

Reply:
What do you think @wulinsheng123? That sounds right to me too.